USAGE.md

```diff
@@ -537,6 +542,29 @@ In order to ease migration from other object stores which do not treat directori
 
 \* Note that in Scenario 4 where the setting is more aggressive than needed, the current behavior is to fall back to creating all intermediate directories. This situation is being revisited in [#414](https://github.com/joyent/java-manta/issues/414)
 
+### Pruning empty parent directories
+
+When `manta.prune_empty_parent_depth`/`MANTA_PRUNE_EMPTY_PARENT_DEPTH` is set to `-1` or a positive value, when deleting an object in Manta, either a file or a directory, the client will attempt to delete parent directories that are empty. If a positive integer is supplied the client will only try to delete up to the number supplied. If -1 is given the client will try to delete in the path until it finds a directory that it can not delete.  
+
+#### Scenario 1 : Prune Empty Parent Depth positive int
+- Given the directory structure : 
+   /Dir1
+   /Dir1/Dir2
+   /Dir1/Dir2/Dir3
+   /Dir1/Dir2/Dir3/test.txt
+   If you have prune_empty_parent_depth set to 1 then delete test.txt, the client should delete Dir3 as well.
+ 
+#### Scenario 2 : Prune Empty Parent Depth -1
+- Given the directory structure : 
+   /Dir1
+   /Dir1/DirA
+   /Dir1/Dir2
+   /Dir1/Dir2/Dir3
+   /Dir1/Dir2/Dir3/test.txt
+   In this case this will if test.txt this will delete test.txt, dir3, and Dir2. It will stop when it gets to the Dir1 since it will not be empty.
```

Comment:
> In this case this will if test.txt this will delete test.txt, dir3, and Dir2.`

=> 

> In this case deletion of `test.txt` will also delete `Dir3`, and `Dir2`.

Reply:
Nope, test.txt is the target, the parent is Dir3, and with the depth of 1 it will only delete Dir3 and not Dir2.

Reply:
Wouldn't the deletion of `Dir3` mean that Dir2 is empty (the scenario says `-1` so I'd assume it would continue up to `Dir1` and stop there because of `DirA`)